### DISCUSSION TITLE
What about FixedUpdate()？

### DISCUSSION THREAD

**Discussion by EsatMan** [stride3d/stride#2401] [Q&A]
Manual did not mention anything about FixedUpdate, is there a Stride version of FixedUpdate() or there isn't? Is the only way to achieve this is something like TaskScheduler or some kind of library?

**Comment by Jklawreszuk** [comment]
Maybe GameBase.IsFixedTimeStep ? (I'm not experienced with Unity tbh)

**Reply by EsatMan** [reply]
Thanks for reply, seems that this is a global configuration, it may cause more problems if I do it wrong. I'd like to learn physics simulation first.

**Comment by Doprez** [comment]
https://github.com/stride3d/stride/pull/1831 

This needs to be documented but this would do what you need AFAIK. You would subscribe to those events and then run your logic within the event calls.

**Reply by EsatMan** [reply]
Thanks for reply, I'll look into it.
